audio/sfx/unreferenced_fanfare.asm

```diff
@@ -1,4 +1,5 @@
-SFX_Unused2_Ch5:
+; This corresponds to SFX_KEY_ITEM in pokegold and pokecrystal.
+SFX_UnreferencedFanfare_Ch5:
```

Comment:
Please make this "UnusedFanfare"; the current Gen 2 convention is to keep `; unreferenced` in comments. (pokered has a couple of those but needs more.)

Reply:
Will do, thanks!